frontend/src/assets/locales/ro/translation.json

```diff
@@ -10,7 +10,9 @@
     "delete": "Sterge",
     "error": {
       "load_entries": "Eroare la incarcarea datelor"
-    }
+    },
+    "cancel": "Anuleaza",
+    "delete_modal_description": "Esti sigur ca doresti stergerea campului din tabel? Aceasta actiune nu este reversibila."
```

Comment:
we should not have this here. And we should be more specific. For instance if we refer to a "division", we should have "Esti sigur ca doresti stergerea acestui rol/departament/filiala?".

Reply:
In this case this should be in "division": {"modal": ...} as described below